### PR TITLE
Add ability to deploy a model to a Kubernetes cluster using the saved schema

### LINKED ISSUE #370: [Deployment] Add ability to deploy a model to a Kubernetes cluster using the saved schema
**Is your feature request related to a problem? Please describe.**
#360 will add the ability to register the schema during the training run. We need have to way currently use that registered schema when deploying a model to a container in a kubernetes cluster.

**Describe the solution you'd like**
Add a new method overload that's not generic to deploy a model to a container and a kubernetes cluster. This method should do the same things as the current one is doing but instead of generating the schema on the fly, it should fetch the stored schema from the run.

### PR DESCRIPTION
**Pull request by aslotte**
## Resolves
Fixes #370 

## Description
Added a method to deploy a model to a cluster that uses the registered model schema instead of passing it in at run time.

### CODE REVIEW

**Comment by aslotte** [comment]
@AnoojNair  / @Brett-Parker this is ready for a review

File: test/MLOps.NET.SQLite.IntegrationTests/KubernetesEndToEndTests.cs [hunk 1]
```diff
@@ -80,6 +81,88 @@ public async Task DeployModel_GivenACompleteRun_ShouldDeployModelToContainer()
...
+
+            var runArtifact = sut.Model.GetRunArtifacts(run.RunId).First();
+
+            var registeredModel = await sut.Model.RegisterModel(run.ExperimentId, runArtifact.RunArtifactId, string.Empty);
+            var deploymentTarget = await sut.Deployment.CreateDeploymentTargetAsync("Test");
+
+            //Act
+            await sut.Deployment.DeployModelToKubernetesAsync(deploymentTarget, registeredModel, string.Empty);
+
+            //Excepts exception
```
Comment (Brett-Parker) [inline test/MLOps.NET.SQLite.IntegrationTests/KubernetesEndToEndTests.cs:163]: Expects exception? Comment needed?

**Review by Brett-Parker — APPROVED** [review]
Minor comment. No need to redo

File: test/MLOps.NET.SQLite.IntegrationTests/KubernetesEndToEndTests.cs (hunk 1 above)
Comment (aslotte) [inline test/MLOps.NET.SQLite.IntegrationTests/KubernetesEndToEndTests.cs:163]:
```suggestion
            //Expects exception
```

**Pull request merged**